Vagrantfile

```diff
@@ -65,31 +66,49 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provision "shell", path: "cookbook/deps/qt5-dependencies.sh"
 
   # Define where to install Qt
-  qtinstallprefix = "/opt/qt-5.7-git"
+  qtinstallprefix = "/opt/qt-5.8-git"
   qmakepath = qtinstallprefix + "/bin/qmake"
   qtcmakepath = qtinstallprefix + "/lib/cmake/"
 
   # Build Qt5 from git
-  config.vm.provision "shell", privileged: false, 
-      args: ["qt5", "5.7", "--prefix=" + qtinstallprefix + " -opensource -confirm-license"],
-      path: "cookbook/build/qt5-git-builder.sh",
-      env: {"MAKEFLAGS" => makeflags}
+  config.vm.provision "shell", privileged: false,
+    path: "cookbook/build/qt5-git-builder.sh",
+    env: {
+      "MAKEFLAGS" => makeflags,
+      "SRC_DIR" => "qt5",
+      "GIT_REPO" => "https://github.com/qt/qt5.git",
+      "GIT_REVISION" => "5.8",
+      "CONFIGURE_ARGS" => "--prefix=" + qtinstallprefix + " -opensource -confirm-license",
+    }
 
   # Build QtIVI
-  config.vm.provision "shell", privileged: false, 
-      args: ["qtivi", "http://code.qt.io/qt/qtivi.git", qmakepath],
-      path: "cookbook/build/qmake-git-builder.sh",
-      env: {"MAKEFLAGS" => makeflags}
+  config.vm.provision "shell", privileged: false,
+    args: ["qtivi", "http://code.qt.io/qt/qtivi.git", qmakepath, "dev", "KEEP_OLD_BUILD"],
+    path: "cookbook/build/qmake-git-builder.sh",
+    env: {
+      "MAKEFLAGS" => makeflags,
+      "SRC_DIR" => "qtivi",
+      "GIT_REPO" => "http://code.qt.io/qt/qtivi.git",
+      "QMAKE_PATH" => qmakepath,
+      "QMAKE_ARGS" => "",
+      "GIT_REVISION" => "465f64cdd9a6f97893e77c9b46bfd1a2f830084d"
+    }
 
   # Build QtApplicationManager
-  config.vm.provision "shell", privileged: false, 
-      args: ["qtapplicationmanager", "http://code.qt.io/qt/qtapplicationmanager.git", qmakepath],
-      path: "cookbook/build/qmake-git-builder.sh",
-      env: {"MAKEFLAGS" => makeflags}
-  
+  config.vm.provision "shell", privileged: false,
+    path: "cookbook/build/qmake-git-builder.sh",
+    env: {
+      "MAKEFLAGS" => makeflags,
+      "QMAKE_PATH" => qmakepath,
+      "SRC_DIR" => "qtapplicationmanager",
+      "GIT_REPO" => "http://code.qt.io/qt/qtapplicationmanager.git",
+      "QMAKE_PATH" => qmakepath,
+      "GIT_REVISION" => "5.8"
+      }
+
   # Install template configuration for QtApplicationManager
   config.vm.provision "shell" do |s|
-    s.inline = "cp -r qtapplicationmanager/template-opt/am /opt/"
+    s.inline = "cp -r qtapplicationmanager/template-opt/am /opt/ || exit 0"
```

Comment:
What's going on here? We're OK with failing to copy this? Can we add a comment about why this is OK?